src/plugins/home/server/services/sample_data/routes/install.ts

```diff
@@ -103,13 +105,20 @@ export function createInstallRoute(
     {
       path: '/api/sample_data/{id}',
       validate: {
-        params: schema.object({ id: schema.string() }),
+        params: schema.object({
```

Comment:
please check your formatting rule settings, seems this line should not be formatted

Reply:
reverted